src/istio/dynamic_encoding/field_encoder.h

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2018 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ISTIO_DYNAMIC_ENCODING_FIELD_ENCODER_H
```

Comment:
`#pragma once` here and elsewhere?

Reply:
I think it's suggested to use #ifndef as compared to #pragma once as #pragma once can be flaky...
I also don't see it's usage in src/istio directory..